tests/test_helpers.py

```diff
@@ -33,13 +33,8 @@
      "20ly_newprojectum\n20sb_firstprojectum\n"
      ),
     (["helper", "l_grants", "-c"],
-     "ALIAS          AWARDNR        ACCOUNT        BEGIN          END            \n"
```

Comment:
to test this one we will have to hand in a date actually, otherwise the test could pass or fail depending on today's date, which is not behavior we want.  It means you will have to put in ``--date`` option.  It is actually useful functionality because I might sometime want to list all the grants that were current last semester or sthg and could pass in ``-c`` and ``--date <sometime in last semester>``